sqldelight-idea-plugin/src/main/resources/META-INF/plugin.xml

```diff
@@ -14,13 +14,6 @@
       features for SQL inside the IDE.
     ]]></description>
 
-  <!-- Plugin's project components -->
-  <project-components>
```

Comment:
this was a major blocker for being installable without restarting intellij, so thats another huge win here